lnrpc/lightning.proto

```diff
@@ -4289,6 +4289,7 @@ message PayReq {
     bytes payment_addr = 11;
     int64 num_msat = 12;
     map<uint32, Feature> features = 13;
+    repeated BlindedPaymentPath blinded_paths = 14;
```

Comment:
since one can ignore this field, it's ok to not be in a major release?

Reply:
it's only breaking changes that we limit to major releases